task/power/src/bsp/cosmo_a.rs

```diff
@@ -345,7 +345,7 @@ impl State {
         .enumerate()
         {
             let (dev, rail) = (builder)(i2c_task);
-            let m = Max5970::new(&dev, rail, Ohms(0.005));
+            let m = Max5970::new(&dev, rail, Ohms(0.005), true);
```

Comment:
It feels really weird to set something like this in two places but I think it's just the way the tweak is working